app/javascript/flavours/glitch/features/local_settings/page/item/index.jsx

```diff
@@ -54,7 +54,7 @@ export default class LocalSettingsPageItem extends React.PureComponent {
       const optionElems = options && options.length > 0 && options.map((opt) => {
         let optionId = `${id}--${opt.value}`;
         return (
-          <label htmlFor={optionId}>
+          <label key={id} htmlFor={optionId}>
```

Comment:
This seems to cause a lot of react warning about duplicate keys

Reply:
Oops, yeah. Fixing that.